static/app/components/events/interfaces/crashHeader/crashActions.tsx

```diff
@@ -46,78 +45,62 @@ const CrashActions = ({
     ? false
     : !!exception?.values?.find(value => value.rawStacktrace) || !!thread?.rawStacktrace;
 
-  const notify = (options: NotifyOptions) => {
-    if (onChange) {
-      onChange(options);
-    }
+  const setStackType = (type: STACK_TYPE) => {
+    onChange?.({stackType: type});
   };
 
-  const setStackType = (type: STACK_TYPE) => () => {
-    notify({stackType: type});
+  const setStackView = (view: STACK_VIEW) => {
+    onChange?.({stackView: view});
   };
-
-  const setStackView = (view: STACK_VIEW) => () => {
-    notify({stackView: view});
-  };
-
-  const getOriginalButtonLabel = () => {
-    if (platform === 'javascript' || platform === 'node') {
-      return t('Original');
-    }
-
-    return t('Symbolicated');
-  };
-
-  const getMinifiedButtonLabel = () => {
-    if (platform === 'javascript' || platform === 'node') {
-      return t('Minified');
-    }
-    return t('Unsymbolicated');
-  };
-
   return (
     <ButtonGroupWrapper>
-      <ButtonBar active={stackView} merged>
-        {hasSystemFrames && (
-          <Button
-            barId={STACK_VIEW.APP}
-            size="xs"
-            onClick={setStackView(STACK_VIEW.APP)}
-            title={
-              hasHierarchicalGrouping
-                ? t(
-                    'The stack trace only shows application frames and frames responsible for grouping this issue'
-                  )
-                : undefined
-            }
-          >
-            {hasHierarchicalGrouping ? t('Most Relevant') : t('App Only')}
-          </Button>
-        )}
-        <Button barId={STACK_VIEW.FULL} size="xs" onClick={setStackView(STACK_VIEW.FULL)}>
-          {t('Full')}
-        </Button>
-        <Button barId={STACK_VIEW.RAW} onClick={setStackView(STACK_VIEW.RAW)} size="xs">
-          {t('Raw')}
-        </Button>
-      </ButtonBar>
+      <SegmentedControl
+        aria-label={t('View')}
+        size="xs"
+        value={stackView}
+        onChange={setStackView}
+      >
+        {[
```

Comment:
Is there a reason for making this an array?

Reply:
`@react-aria` doesn't consider `false` a valid collection child — passing `false` won't cause an error but will result in a TS warning — so I had to use an array to avoid a warning. I considered updating `SegmentedControl`'s prop types to allow for boolean children, but decided against it because it would involve some type coercion. It seems that I underestimated how often we'd want to conditionally add/remove items with `&&` though, so lemme try it again.

Reply:
The other option is you can do `{shouldRender ? <Stuff /> : null}`